package.json

```diff
@@ -20,7 +20,6 @@
     "@adobe/eslint-config-helix": "1.1.0",
     "@adobe/helix-testutils": "0.3.0",
     "@adobe/jsonschema2md": "4.0.13",
-    "@pollyjs/adapter-fetch": "4.0.2",
```

Comment:
Remnants from the hackathon but not needed since `adapter-node-http` already intercepts fetch